fairscale/utils/parallel.py

```diff
@@ -58,18 +58,45 @@ def enable_pytorch_sync_bn(module: torch.nn.Module) -> None:
             layer._specify_ddp_gpu_num(1)  # type: ignore
 
 
-def get_global_group() -> None:
+def get_process_group_cached(ranks: Optional[List[int]] = None) -> ProcessGroup:
```

Comment:
nit: can we rename this to "get_cached_process_group" instead?

Reply:
Do you have a strong preference? I like get_process_group to be together more and not broken by "cached". If you don't insist I'd like to keep it.

Reply:
It is a "nit"  hence you are welcome to keep it as is :)  I think it made more sense for me since the process group we are getting is described by the "cached" string.